docs/naps/8-multiple-canvases.md

```diff
@@ -0,0 +1,261 @@
+
+## Motivation and Scope
+The ability to view n-D data from multiple perspectives is a common feature request, and has proved useful in many other tools for data exploration and analysis. Here is a sampling of issues requesting support and discussing potential implementations:
+
+* [#5348](https://github.com/napari/napari/issues/5348) Multicanvas viewer
+* [#2338](https:////github.com/napari/napari/issues/2338) Multicanvas API Thoughts
+* [#760](https:////github.com/napari/napari/issues/760) Linked multicanvas support
+* [#662](https:////github.com/napari/napari/issues/662) Linked 2D views
+* [#561](https:////github.com/napari/napari/issues/561) multicanvas grid display for layers in Napari
+* [#1478](https:////github.com/napari/napari/issues/1478) Orthogonal viewer plugin
+
+Several plugins and examples have been created to address these limitations, for example:
+* [napari-3d-ortho-viewer](https://github.com/gatoniel/napari-3d-ortho-viewer/tree/main)
+* [multiple viewer widgets example](https://napari.org/stable/gallery/multiple_viewer_widget.html#sphx-glr-gallery-multiple-viewer-widget-py)
+
+This document is intended to cover what [#5348](https://github.com/napari/napari/issues/5348) refers to as "True Multicanvas". The changes to "Grid Mode" as described in that issue may be an important step along the way. This should be considered as part of the [implementation plan](#Implementation-Plan).
+
+Providing native support in napari would allow developers to more easily create these experiences, enable interoperability between such plugins, and improve performance.
+
+### Out of Scope
+* Improvements to VisPy to support multiple views of the same `SceneGraph` (sharing data, saving VRAM) - for relevant discussion start with [vispy/#1992](https://github.com/vispy/vispy/issues/1992).
+* For now, canvas arrangement (for example: tiling behavior) will be handled in the view only (left to Qt or custom Qt widgets). Making this state (de)serializable is out of scope for this project, but may be relevant when implementing a “savable viewer state” feature.
+* Normalizing slice data for different layer types, though this may benefit in the course of this work.
+* Specific UI implementations will be explored as part of this work, but I expect UX and UI will be formalized later (possibly in a separate NAP).
+* Supporting alternative frontend (Qt) and backend (Vispy) frameworks. While this work should not make such tasks more difficult in the future, explicit consideration is out-of-scope until further progress is made in these areas.
+* [Non-goals also in NAP-3](https://napari.org/dev/naps/3-spaces.html#non-goals) are related but also considered out-of-scope here
+    * Separation of rendering information from the Layer(Data) model
```

Comment:
I think layergroups here could potentially provide a cleaner API, but overall the API is similar to current grid mode at least for multichannel view, thus could be an orthogonal effort.

Reply:
I don't think it's entirely orthogonal, but perhaps mostly. I think it depends on the scope of features we want. For example just below I propose for all canvases to share a common layer list, with independent slicing, camera, dimensionality, and layer visibility. I think this would be more relevant if supporting (for example) different colormaps for the same data in two canvases.